bench.js

```diff
@@ -0,0 +1,75 @@
+var Graylog = require('.').graylog;
+var fs = require('fs');
+var client;
+var servers = [
```

Comment:
Would suggest to make this accept CLI arguments, so that you can test against an existing Graylog setup